docs/UserGuide.md

```diff
@@ -12,23 +12,22 @@ A good example is students who are struggling to do well in both their studies a
 
 FriendDex uses a command line interface; this means that you type commands to interact with the application. 
 
-To help you get started, you should read the [Installation](#installation) and [Quick start](#quick-start) section first.
+To help you get started, we recommend reading the [Installation](#installation) and [Quick Start](#quick-start) sections first.
 
 * Table of Contents
 {:toc}
 
 --------------------------------------------------------------------------------------------------------------------
 ## About
-This document can be thought of as a manual and a reference guide for FriendDex. It will guide you on how to use FriendDex and will provide complete information on each available command.
-Furthermore, the guide gives information on the User Interface (UI) and the other useful features of FriendDex. Each section of the guide can be read independently. 
-You can view the full list of content using the Table of Contents above. You can also use your document viewer’s Find function to quickly navigate to the content you want to know more about. 
+This document can be thought of as a manual and reference guide for FriendDex. It will guide you on how to use FriendDex and will provide complete information on all available commands.
+Furthermore, the guide gives information on the User Interface (UI) and the other useful features of FriendDex. Each section of the guide can be read independently.
+For an outline of the content in this document, refer to the Table of Contents above. You may also use your document viewer’s Find function to quickly navigate to the content you want to know more about.
 
-It is generally advised for new users to at least read through the [Quick Start](#quick-start) section to familiarise themselves with FriendDex.
+It is generally advisable for new users to visit the [Quick Start](#quick-start) section first to familiarise themselves with FriendDex.
 
-Note the following symbols and formatting used in this document:
+### Symbols
 
-`list n/Close Friends` <br>
```

Comment:
Can add the markup portion here for CS2101?

Reply:
Okay but need to reword. The current description is inconsistent as just one section later we use it for "Java `11`".